mParticle-iterable/MPKitIterable.m

```diff
@@ -0,0 +1,117 @@
+//
+//  MPKitIterable.m
+//
+//  Copyright 2016 mParticle, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "MPKitIterable.h"
+
+// This is temporary to allow compilation (will be provided by core SDK)
+NSUInteger MPKitInstanceIterable = 1003;
+
+@interface MPKitIterable() {
+    NSURL *clickedURL;
+}
+
+@end
+
+@implementation MPKitIterable
+
+/*
+    mParticle will supply a unique kit code for you. Please contact our team
+*/
++ (NSNumber *)kitCode {
+    return @1003;
```

Comment:
@samdozor: All the examples I see hardcode the return value. Let me know how I should change this.